tests/llmcompressor/transformers/obcq/recipes/quant_and_sparse.yaml

```diff
@@ -13,12 +13,13 @@ test_stage:
           weights:
             num_bits: 8
           targets: ["Linear"]
+    GPTQModifier:
```

Comment:
QuantizationModifier and GPTQModifier should be in order, otherwise we end up in a situation were the "naive" minmax quantization is used for the first sparsity calculation, then the more accurate GPTQ quantization is applied afterwards. We want sparsity and quantization to be fully separate operations, not apply sparsity in the middle of the algorithm